pkg/github/raw_file_service.go

```diff
@@ -0,0 +1,24 @@
+package github
+
+import (
+	"fmt"
+
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/arquillian/ike-prow-plugins/pkg/utils"
+)
+
+// RawFileService encapsulates retrieval of files in the given GitHub repository change
+type RawFileService struct {
+	Change scm.RepositoryChange
```

Comment:
If we make a `RepositoryChange` as a part of this struct we might want to extend the doc saying it represents the file in the given state of repository indicated by hash.

Reply:
in the same way as we do it for the config file - make sense, I'll do that

Reply:
1956e06